src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,48 @@
+.calendar {
+  display: flex;
+  margin: auto;
+  gap: $cellGap;
+  flex-wrap: wrap;
+  width: calendarWidth($cellSize, $weekLength, $calendarPadding, $cellGap);
+  padding: $calendarPadding;
+
+  &__cell {
+    display: flex;
+    width: $cellSize;
+    height: $cellSize;
```

Comment:
width and height are the same. So you can create a mixin `square` where you pass just one `size`